virttest/guest_agent.py

```diff
@@ -732,3 +732,31 @@ def fsthaw(self, check_status=True):
                     raise
             return ret
         return -1
+
+    def file_operate(self, operate, **kwargs):
```

Comment:
`guest_file_operation((self, operation, **kwargs))` or `operate_guest_file((self, operation, **kwargs))` would be more meaningful.

Reply:
Better use `file_open`, `file_write` to wrap up `file_operate` so it will be easy for user to use.
Leaving a function with `**kwargs` is in this situation a really hard for users to use.